app/controllers/admin_general_controller.rb

```diff
@@ -12,25 +12,26 @@ def index
     @requires_admin_requests = InfoRequest.find_in_state('requires_admin')
     @error_message_requests = InfoRequest.find_in_state('error_message')
     @attention_requests = InfoRequest.find_in_state('attention_requested')
-    @blank_contacts = PublicBody.
-      includes(:tags, :translations).
-        where(:request_email => "").
-          order(:updated_at).
-            select { |pb| !pb.defunct? }
-    @old_unclassified = InfoRequest.find_old_unclassified(:limit => 20,
-                                                          :conditions => ["prominence = 'normal'"])
-    @holding_pen_messages = InfoRequest.
-      includes(:incoming_messages => :raw_email).
-        holding_pen_request.
-          incoming_messages
-    @new_body_requests = PublicBodyChangeRequest.
-      includes(:public_body, :user).
-        new_body_requests.
-          open
-    @body_update_requests = PublicBodyChangeRequest.
-      includes(:public_body, :user).
-        body_update_requests.
-          open
+    @blank_contacts = PublicBody
```

Comment:
We've been using the [trailing `.`](https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains) multi-line method chaining style elsewhere.